PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +25,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((double)numberCorrectAnswers / (double)numberOfQuestions * 100) + "%";
```

Comment:
Only really needed to cast one of these, but nothing wrong with doing both. Only suggestion I would make would be to round the answer at the end in case you don't get a whole number in the result.
```suggestion
return Math.Round((float)numberOfCorrectAnswers / numberOfQuestions * 100) + “%”
```